guide/git_flow/bibliography.md

```diff
@@ -73,6 +73,11 @@ Online courses and step-by-step demonstrations explaining the most popular Git c
 ### GitHub
 #### Pull Requests (PR)
 ```{eval-rst}
+.. [quick_rundown_video] 
+   freeCodeCamp Youtube Video to understand basic UI of GitHub
```

Comment:
```suggestion
   FreeCodeCamp Youtube Video to understand basic UI of GitHub
```
Please capitalize the start of the sentence (Unless the company is actually called `freeCodeCamp` and the lower "f" is intentional.

Reply:
The "f" is lower case because it is a specific name.

I will add bullet points in the coming week about the video, as I watch it again.